docs/reference/bare_metal/c_small_libs.md

```diff
@@ -0,0 +1,55 @@
+<h1 id="using-small-c-libraries">Using small C libraries in Mbed OS</h1>
```

Comment:
Why is that HTML header rather than markdown markup?

Reply:
It's a unique header ID. We use it in certain circumstances across products, such as to prevent duplication of URLs, to shorten URLs and when we want the URL to be something other than the header.